packages/gated-content/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lens-protocol/gated-content",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
@desfero what do you think we should do here? It seems ok to me to have this package live its own versioning until we reach v1.0.0, which will re-sync all of them.

Reply:
it's a dependency of `react` and `react-web`.